### PR TITLE
ui(stats): format stat numbers w/ commas

### PR DESCRIPTION
**Pull request by djkirby**
Screen Shot 2019-05-24 at 8 30 02 AM

### CODE REVIEW

File: app/javascript/components/widgets/numbers/carousel.jsx [hunk 1]
```diff
@@ -32,19 +33,24 @@ const getEventCounts = gql`
...
 class SubscribedEvents extends React.Component {
   componentDidMount() {
     this.props.subscribeToPublishedEvents();
   }
 
   render() {
     const { githubPull, githubCommit, slackMessage } = this.props;
+    const githubPullCount = numeral(githubPull).format(STAT_FORMAT);
+    const githubCommitCount = numeral(githubCommit).format(STAT_FORMAT);
+    const slackMessageCount = numeral(slackMessage).format(STAT_FORMAT);
```
Comment (aaddaamm) [inline app/javascript/components/widgets/numbers/carousel.jsx]: what is the reason for using this numeral library? could we accomplish this without adding the dependency?

Comment (djkirby) [reply app/javascript/components/widgets/numbers/carousel.jsx]: 🤷‍♂ probably but that's more code we'd have to maintain, I'd rather let numeral do the formatting personally

Comment (aaddaamm) [reply app/javascript/components/widgets/numbers/carousel.jsx]: could we use this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#Using_toLocaleString

Comment (cbarber) [reply app/javascript/components/widgets/numbers/carousel.jsx]: I'm ok with 3.8kB added for an expressive way to format numbers
https://bundlephobia.com/result?p=numeral@2.0.6

**Pull request merged**